doc/rados/operations/placement-groups.rst

```diff
@@ -171,6 +171,10 @@ You can set the minimum number of PGs for a pool with::
 
   ceph osd pool set <pool-name> pg_num_min <num>
 
+You can set the maximum number of PGs for a pool with::
+
+  ceph osd pool set <pool-name> pg_num_max <num>
```

Comment:
@liewegas how do you think we should handle existing clusters? 

We could have the devicehealth module set this for its pool (and ignore errors in case the osds haven't upgraded yet) every 10 seconds until it succeeds

Reply:
Yeah, devicehealth's main loop can just check the current pool parameters and the osdmap release and update if necessary + possible